docs/sphinx/install.rst

```diff
@@ -1,122 +1,170 @@
 
-Use CUDA Quantum in a Terminal
-+++++++++++++++++++++++++++++++++++++
+Once you have downloaded an image, the container can be run using the command
 
-The container can be run using the following command
+.. code-block:: console
+
+    docker run -it --name cuda-quantum nvcr.io/nvidia/nightly/cuda-quantum:latest
+
+Replace the image name and/or tag in the command above, if necessary, with the one you want to use.
+This will give you terminal access to the created container. To enable support 
+for GPU-accelerated backends, you will need to pass the :code:`--gpus` flag when launching
+the container, for example:
 
 .. code-block:: console
 
-    docker run -it --name cuda-quantum <image_name>
+    docker run -it --gpus all --name cuda-quantum nvcr.io/nvidia/nightly/cuda-quantum:latest
+
+.. note:: 
 
-replacing :code:`<image_name>` with the name and tag of the image you downloaded.
+  This command will fail if you do not have a suitable NVIDIA GPU available, or if your driver 
+  version is insufficient. To improve compatibility with older drivers, you may need to install the 
+  `NVIDIA container toolkit`_.
 
-This will give you terminal access to the created container, for example
+.. _NVIDIA container toolkit: https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html
 
-.. code-block:: console 
+You can stop and exit the container by typing the command :code:`exit`. If you did not specify
+:code:`--rm` flag when launching the container, the container still exists after exiting, as well as any 
+changes you made in it. You can get back to it using
+the command :code:`docker start -i cuda-quantum`. 
+You can delete an existing container and any changes you made using :code:`docker rm -v cuda-quantum`.
 
-    user@host:~$ docker run -it --name cuda-quantum nvcr.io/nvidia/nightly/cuda-quantum:latest
-    To run a command as administrator (user "root"), use "sudo <command>".
-    See "man sudo_root" for details.
+When working with Docker images, the files inside the container are not visible outside the container environment. 
+To facilitate application development with, for example, debugging, code completion, hover information, and so on, 
+please take a look at the section on :ref:`Development with VS Code <docker-in-vscode>`.
 
-    =========================
-    NVIDIA CUDA Quantum
-    =========================
+Alternatively, it is possible, but not recommended, to launch an SSH server inside the container environment and connect an IDE using SSH. To do so, make sure you have generated a suitable RSA key pair; if your `~/.ssh/` folder does not already contain the files `id_rsa.pub` and `id.rsa`,
+follow the instructions for generating a new SSH key on `this page <https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent>`__.
+You can then launch the container and connect to it via SSH by executing the following commands:
```

Comment:
For being not recommended, we have a lot of instructions (albeit perhaps optional instructions) in this guide that depend on them doing this. Perhaps we should strike that clause and just say `Alternatively, it is possible to launch`...?

Reply:
We don't have any instructions in the rest of the guide that depend on doing this. Could you clarify what gave you this impression? Also, did you read the md start to finish in one go, or did you look at the docs and follow the links that someone with different preferences would/should follow?

Reply:
Maybe I'm missing a subtle difference, but I think this is being recommended in lines 242-275, where it talks about configuring and running an SSH server inside your container, no?

(I was not looking at the docs preview as I don't think has completed yet, but I'll take a look at it once it's ready.)

Reply:
Yes, if you run a singularity container, VS Code does not have any support for doing this for you and you indeed need to manually run an ssh server. But this is why I was asking if you read this "like a user" (i.e. probably skim and jump to the sections that seem relevant for your scenario) or "like a code reviewer" (read start to finish and see similarities). It seems unlikely that anyone would read the section about Docker and Singularity; if they are ok with Docker and follow the links the page directs them to, they should have a working setup without ever coming across it. If they cannot use Docker, they are directed to the section on Singularity early, making it clear that they don't/should continue reading about something they won't use.